dbms/src/Flash/Mpp/MppVersion.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <string>
+
+namespace DB
+{
+enum MppVersion : int64_t
```

Comment:
Rename to MPPVersion and rename the file name also?

Reply:
I'm not sure whether there is certain rule about naming mpp. There is also a directory named `dbms/src/Flash/Mpp/`.